src/com/xilinx/rapidwright/rwroute/RouteNode.java

```diff
@@ -51,17 +51,17 @@ abstract public class RouteNode implements Comparable<RouteNode> {
 
     /** The associated {@link Node} instance */
     protected Node node;
-    /** The type of a rnode*/
-    private RouteNodeType type;
     /** The tileXCoordinate and tileYCoordinate of the INT tile that the associated node stops at */
     private short endTileXCoordinate;
     private short endTileYCoordinate;
     /** The wirelength of a rnode */
     private short length;
```

Comment:
Is there any gain in making `length` a byte instead of a `short`? More generally, I think the longest (non clock) wire we have is a SLL which is 60 tiles. I'm unclear as to where there are a longer clock wires, but in any case, I don't believe clock routing uses the `RouteNode` class anyway.

If you do reduce it to a `byte`, let's add an assertion before setting it to ensure this invariant doesn't get tripped in the future.